kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -334,6 +337,25 @@
       updateContentState(contentState, forceSave = true) {
         this.updateContentNodeState({ contentState, forceSave });
       },
+      navigateToRegularContext(message) {
+        let id = message.nodeId;
+        return ContentNodeResource.fetchModel({ id })
+          .then(contentNode => {
+            let routeBase, path;
+            if (contentNode && contentNode.kind !== 'topic') {
+              routeBase = '/topics/c';
```

Comment:
I would prefer to a route object here. this route is equivalent to `{ name: 'TOPICS_CONTENT' m ...}`, I think (see the router file in the learn plugin).